.travis.yml

```diff
@@ -48,3 +48,10 @@ stages:
  - test
  - name: deploy
    if: branch = master AND NOT type = pull_request
+env:
+  global:
+  - secure: CVgUFV84mm7Jmi3PFOh4OWNEa5NzIBOSLEQys6gF5eb4Y36cCtbfWrKr2bBC5ZvCg0yFvlc4xhk4Q2InTN4jXBKolNf9OAwfuEDsFveiQ2UJI1+5gL+P84DXpkEeTKB1x6nFFUWmvXqe2juw3rtq9i8SZrSUV8zL63rUKkry1yoRvgsSevGRSXL8p3KCi68HWax+oAbMUPPn4Lvc1xToZGGPHSe8dJV0g/Lty24266IbXdJyHYG0ylYd41FiJpeMJ7QREl+CkdYi4DnUEE3ZUTBVYRn2gTebgtuay7tyhA70TbYhmkxprSrQvm10VgOuH0Nab+xZqBR+UosAZsGzq9K2dgc9CMZ3oVjft0K6MKmDBUdH2L5WKODUgV1/4ZwuW+lSsEYUFH4jBXihveS1k0+mPShc3m/F7s4olUcX8050fAB+ctm4E8detyBJdiqdcnBIOaQWm/eaXdHPaCIiMG4dMMMdrA+rXmqIsk6UtO76Qsuhy7mEJyhGkwrVqGRlI5okqGRBasLhzPgVx4w1fXinYe0ZgF1URGAHdTVY0lZP5ePTgnRlICkkeVBnjtsgxv20RVevGcgomWzDwnO8R9GQ+zOrv7UcK/5rJfcH5wYkG2OR6ZFRRidglQKqYzGDdzrXkOCwua8FiiMc+TmNx34c5AXvtRE5WkWbzbC1Jdc=
```

Comment:
I recommend to specify with a comment to which key it corresponds. it will allow to regenerate a single one at a time and know which one are defined.

Reply:
I was trying this out but looks like the build still failed, sigh